tests/test-cases/Group9-VIC-Admin/9-2-VICAdmin-CertAuth.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation  Test 9-2 - VICAdmin CertAuth
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  certs=${true}
+Suite Teardown  Cleanup VIC Appliance On Test Server
+Default Tags
+
+*** Test Cases ***
+Wait
+    Run  sleep 30
+
+Display HTML
+    ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}
```

Comment:
From a technical standpoint it's tricky because the Go HTTP module is handling acceptance and verification of certificates, so it's auth turned on everywhere or nowhere. 

This is something that can be done, and the way to do it involves the same mechanisms as username/password authentication -- I can swap the HTTP server to `tls.VerifyClientCertIfGiven` and write a handler wrapper that displays status if no certificate is given & protects all of the routes if it is not.